extensions/Postgres/Postgres/Internals/PostgresDbClient.cs

```diff
@@ -28,7 +28,7 @@ internal sealed class PostgresDbClient : IDisposable
     private const string PgErrDatabaseDoesNotExist = "3D000"; // invalid_catalog_name
 
     private readonly ILogger _log;
-    private readonly NpgsqlDataSource _dataSource;
+    private readonly NpgsqlDataSourceBuilder _dataSourceBuilder;
```

Comment:
by removing `NpgsqlDataSource` we can also remove `IDisposable` (line 22) and the `Dispose` methods

Reply:
I have removed the `IDsposable` interface from `NpgsqlDataSource `, that in turn makes the `IDisposable` interface on `PostgresMemory` on longer necessary.